src/jvm/main/org/jetbrains/kotlinx/lincheck/transformation/TransformationUtils.kt

```diff
@@ -347,3 +347,5 @@ internal const val ASM_API = Opcodes.ASM9
 internal val STRING_TYPE = getType(String::class.java)
 internal val CLASS_TYPE = getType(Class::class.java)
 internal val CLASS_FOR_NAME_METHOD = Method("forName", CLASS_TYPE, arrayOf(STRING_TYPE))
+
+internal fun isClassLoader(className: String): Boolean = className.contains("ClassLoader")
```

Comment:
Please rename the function to indicate a possibility of false positives (or make it `isNotClassLoader` and add the comment about false positives where you use the function)